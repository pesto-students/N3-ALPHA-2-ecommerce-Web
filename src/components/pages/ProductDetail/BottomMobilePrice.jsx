import React from 'react';
import { useTranslation } from 'react-i18next';

const MobileBottomPrice = ({ price = 0, isInCart = false, handleClick }) => {
    const { t } = useTranslation();
    return (
        <div className="bottomPrice">
            <h2 className="bottomPrice_price">
                {`₹${price}`}
                <span>
                    <del>{`₹${parseInt((110 / 100) * price)}`}</del>
                </span>{' '}
            </h2>
            <button
                class="bottomPrice_btn"
                onClick={(e) =>
                    handleClick(isInCart ? 'checkout' : 'addToCart')
                }
            >
                {isInCart ? t('checkout_text') : t('addCart_text')}
            </button>
        </div>
    );
};

export default MobileBottomPrice;
