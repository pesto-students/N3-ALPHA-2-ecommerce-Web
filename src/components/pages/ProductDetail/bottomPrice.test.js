import renderer from 'react-test-renderer';
import BottomMobilePrice from './BottomMobilePrice';

test('Product Detail Snapshot', () => {
    const tree = renderer.create(<BottomMobilePrice />).toJSON();
    expect(tree).toMatchSnapshot();
});
