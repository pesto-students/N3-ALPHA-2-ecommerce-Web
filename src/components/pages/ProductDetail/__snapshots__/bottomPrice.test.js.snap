// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product Detail Snapshot 1`] = `
<div
  className="bottomPrice"
>
  <h2
    className="bottomPrice_price"
  >
    ₹0
    <span>
      <del>
        ₹0
      </del>
    </span>
     
  </h2>
  <button
    class="bottomPrice_btn"
    onClick={[Function]}
  >
    addCart_text
  </button>
</div>
`;
